.gitignore

```diff
@@ -66,3 +66,14 @@ blockchain_nodes/bitcoin/
 blockchain_nodes/parity/parity
 .yalc
 yalc.lock
+
+# Default ignored files
```

Comment:
This kind of stuff can easily go in your home `.gitignore`

Reply:
I would assume this was just copy pasted from nectar though?

Reply:
dropped that commit

Reply:
yep it was copy pasted from nectar @thomaseizinger